telco-examples/edge-clusters/dhcp/eib/custom/scripts/04-sriov.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# copy the artifacts
+cp sriov-auto-filler.sh /var/sriov-auto-filler.sh
```

Comment:
I might have missed this but where is this script used?

Reply:
telco-examples/edge-clusters/airgap/eib/custom/files/sriov-auto-filler.sh